tests/tests/optionsTests.robot

```diff
@@ -114,4 +110,20 @@ Enable Debug Output functionality test
     Check If Log Console Contains    loaded
     Wait Until Element Is Visible    ${ReloadBadge}
 
-
+Verify Output log to a file option
+    [Documentation]    Verify Output log to a file option
+    [Tags]    options
+    ${downloaded_file_path}    Set Variable    ${DOWNLOAD_PATH}${file_4mb}[archive_name].zip
+    Click Add Files Button
+    Ubuntu Upload Test File    ${file_4mb}[path]
+    Click Load Button
+    Check If Log Console Contains    Opening "${file_4mb}[name]"
+    Validate Output Description    ${file_4mb}[archive_name]
+    Click Element    ${OptionsButton}
+    Wait Until Element Is Visible    ${LogsButton}
```

Comment:
At this point it would be more useful to wait for ${OutputLogsSwitch} element since it will be use in next step.

Reply:
Done